Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/remove-addon.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pre-delete-admin
+  namespace: kube-system
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "20"
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pre-delete-agent
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "20"
+rules:
+- apiGroups: ["apps"]
+  resources: ["deployments"]
+  verbs: ["delete"]
```

Comment:
Does this means azure-policy account can remove any deployment? Would this be fine for customer? Can we confirm on this?

Reply:
Only the hook has permissions to delete deployments. It uses a separate service account to ensure the add-on does not have these permissions